src/frontends/paddle/src/op/reverse.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/opsets/opset1.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+
+using namespace default_opset;
+
+NamedOutputs reverse(const NodeContext& node) {
+    auto x = node.get_input("X");
+    // axis is a vector
+    auto axis = node.get_attribute<std::vector<int32_t>>("axis");
+    const auto rank = x.get_partial_shape().rank();
+
+    if (rank == 1) {
+        auto axis_node = default_opset::Constant::create(ngraph::element::i32, Shape{1}, axis);
+        auto reverse_op = std::make_shared<ov::opset1::Reverse>(x, axis_node, opset1::Reverse::Mode::INDEX);
+        auto out = node.default_single_output_mapping({reverse_op}, {"Out"});
+        return out;
+    } else {
+        // get value from axis
+        auto seq_axis = axis.at(0);
+        // batch_axis
+        int64_t batch_axis = !seq_axis;
```

Comment:
if seq_axis == 0 get batch_axis == 1, is it expected?

Reply:
> if seq_axis == 0 get batch_axis == 1, is it expected?

remove the useless codes because the ov reverse ir can map paddle completely.